charts/aerospike/templates/configmaps.yaml

```
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: aerospike-config
+data:
+  aerospike.conf: |
+    # Aerospike database configuration file
+    # This template sets up a single-node, single namespace developer environment.
+    #
+    # Alternatively, you can pass in your own configuration file.
+    # You can see more examples at
+    # https://github.com/aerospike/aerospike-server/tree/master/as/etc
+
+    # This stanza must come first.
+    service {
+
+    }
+
+    logging {
+            # Send log messages to stdout
+            console {
+                    context any info
+            }
+    }
+
+    network {
+            service {
+                    address any
+                    access-address aerospike-0
```

Should this be the FQDN? 
```suggestion
                    access-address aerospike-0.aerospike.default.svc.cluster.local
```

I don't know, but my gut says it might

---

Trying it out with this change now, looks like it is working as expected, so I'll keep the change.